docs/publish/publish-add-in-vs-code.md

```diff
@@ -41,12 +41,70 @@ When your add-in is working as desired, you can publish it directly through Visu
 
     When the build completes, the **dist** folder in the root directory of your add-in project will contain the files that you'll deploy in subsequent steps.
 
-10. To deploy, select the Files explorer, select and hold (right-click) your **dist** folder, and choose **Deploy to Static Website**. When prompted, select the storage account you created previously.
+10. To deploy, select the Files explorer, select and hold (right-click) your **dist** folder, and choose **Deploy to Static Website via Azure Storage**. When prompted, select the storage account you created previously.
 
-![Deploying to a static website.](../images/deploy-to-static-website.png)
+    ![Deploying to a static website.](../images/deploy-to-static-website.png)
 
 11. When deployment is complete, a **Browse to website** message appears which you can select to open the primary endpoint of the deployed app code.
 
+## Deploy custom functions for Excel
+
+If your add-in has custom functions there are a few more steps to enable them on the Azure Storage account. First you need to enable CORS so that Office can access the functions.json file.
```

Comment:
Missing comma.
```suggestion
If your add-in has custom functions, there are a few more steps to enable them on the Azure Storage account. First you need to enable CORS so that Office can access the functions.json file.
```